test/infrastructure/docker/internal/docker/loadbalancer.go

```diff
@@ -101,7 +101,11 @@ func getLoadBalancerImage(dockerCluster *infrav1.DockerCluster) string {
 
 // ContainerName is the name of the docker container with the load balancer.
 func (s *LoadBalancer) containerName() string {
-	return fmt.Sprintf("%s-lb", s.name)
+	lbName := s.name
+	if len(s.name) > 57 {
+		lbName = s.name[:57]
+	}
+	return fmt.Sprintf("%s-lb", lbName)
```

Comment:
Let's comment why are we doing this for future reference.
Also, should we do the same for the containers hosting CP and workers?

Reply:
Yeah we need to - but it's a little bit more complicated. I'm working on a fix there.